src/ledger/LedgerManagerImpl.cpp

```diff
@@ -529,26 +529,20 @@ LedgerManagerImpl::getSorobanNetworkConfig()
 {
     releaseAssert(hasSorobanNetworkConfig());
 #ifdef BUILD_TESTS
-    if (mApp.getConfig().MODE_USES_IN_MEMORY_LEDGER)
-    {
-        return *mSorobanNetworkConfig;
-    }
-    else
+    return *mSorobanNetworkConfig;
```

Comment:
I'm not particularly happy with this change, but it looks like our tests are still modifying soroban configs directly without closing ledgers, so I'm not sure what a better alternative would be. @SirTyson maybe you have some input?